src/Moq/Properties/Resources.resx

```diff
@@ -348,4 +348,7 @@ Expected invocation on the mock once, but was {4} times: {1}</value>
 		<value>The following invocations on mock '{0}' were not verified:
 {1}</value>
 	</data>
-</root>
\ No newline at end of file
+	<data name="CallBaseUsedOnDelegateException" xml:space="preserve">
```

Comment:
Minor suggestion: Rename `CallBaseUsedOnDelegateException` to `CallBaseCannotBeUsedWithDelegateMocks`. (The resource names usually reflect its contents, not what it will be used for.)